staubli_rx160_support/urdf/rx160l.urdf

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" ?>
```

Comment:
Also: all links/joints have a `staubli_` prefix added: is that intentional? It is not present in either the `_macro.xacro` or the top-level `.xacro`.